src/chrome/content/rules/Pochtabank.ru.xml

```diff
@@ -0,0 +1,10 @@
+<!--(www.)? protocol error-->
```

Comment:
WFM.
Please also search for other subdomains, there are a lot more supporting https.